.jenkins/pytorch/build.sh

```diff
@@ -159,6 +159,21 @@ if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
   python tools/amd_build/build_amd.py
   python setup.py install --user
 
+  # remove sccache wrappers post-build; runtime compilation of MIOpen kernels does not yet fully support them
```

Comment:
Can you please guard this block with `if [[ $BUILD_ENVIRONMENT == *rocm* ]]; then`?

Reply:
Yep, I considered that -- it's already inside such an if block.

Reply:
Block starts above on line 118.